public/index.html

```diff
@@ -12,6 +12,7 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Heebo" rel="stylesheet">
+    <link rel="stylesheet" href="./card.css">
```

Comment:
Why is this necessary? Shouldn't this be styled components? Even if you're forced to have styled classes, keep them to the SC reset file rather than a css file.

Reply:
Removed :wastebasket: